src/main/resources/migration/V20170208000000__add_parent_account.sql

```diff
@@ -0,0 +1,3 @@
+alter table analytics_accounts add parent_account_id varchar(36) default null after account_external_key;
+alter table analytics_accounts add parent_account_name varchar(100) default null after parent_account_id;
+alter table analytics_accounts add parent_account_external_key varchar(50) default null after parent_account_name;
```

Comment:
Done in https://github.com/killbill/killbill-analytics-plugin/commit/f097ca4458b07e05e94bf2f2cb09a4e9878ab137.